tools/Dockerfile

```diff
@@ -131,15 +130,17 @@ RUN tar -zxvf /tmp/conftest_${CONFTEST_VERSION}_Linux_x86_64.tar.gz -C /tmp && m
 ADD https://github.com/plexsystems/konstraint/releases/download/v${KONSTRAINT_VERSION}/konstraint-linux-amd64 /tmp
 RUN mv /tmp/konstraint-linux-amd64 konstraint && chmod +x konstraint
 
-#dyff 
-ADD https://github.com/homeport/dyff/releases/download/v${DYFF_VERSION}/dyff_${DYFF_VERSION}_linux_amd64.tar.gz /tmp
-RUN tar -zxvf /tmp/dyff_${DYFF_VERSION}_linux_amd64.tar.gz -C /tmp && mv /tmp/dyff dyff
-
 # node
 RUN curl -sL https://deb.nodesource.com/setup_${NODE_VERSION}.x | bash -
 RUN apt-get install -y nodejs && \
   npm install -g ajv-cli@v3.3.0 json-dereference-cli@0.1.2 zx
 
+# docker
+RUN curl -sSL https://get.docker.com/ | sh
+
+# kind
```

Comment:
I see this is copied over from `kind/Dockerfile`. That file still has these statements.

Reply:
that other Dockerfile will be removed

Reply:
but I reverted the kind changes like you suggested...and also because I don't wish to add dev tools to the final core container